### PR TITLE
add database columns and config for feature flag

### PR DESCRIPTION
**Pull request by mikehelmick**
forst towards: #1567

## Proposed Changes

* Add DB column for realm auto verification certificate key rotation
* Add config feature flag that will control this - default to OFF at a system level

**Release Note**

```release-note
Adds database column for realm level auto key rotation settings.
```

### CODE REVIEW

File: pkg/database/migrations.go [hunk 1]
```diff
@@ -1968,6 +1968,16 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 					`DELETE FROM users WHERE deleted_at IS NOT NULL`)
 			},
 		},
+		{
+			ID: "00086-AddRealmAutoRotateSetting",
+			Migrate: func(tx *gorm.DB) error {
+				return tx.AutoMigrate(&Realm{}).Error
```
Comment (whaught) [inline pkg/database/migrations.go:1974]: if it's just the one bool column, maybe we should just do the standard `ALTER TABLE` dance. I feel like automigrate has screwed us up before

Comment (whaught) [reply pkg/database/migrations.go:1974]: Also do we want the column to be NOT NULL?

**Review by whaught — APPROVED** [review]

File: pkg/database/migrations.go (hunk 1 above)
Comment (mikehelmick) [reply pkg/database/migrations.go]: removed auto migrate and added not null.

**Review by whaught — APPROVED** [review]

**Pull request merged**